lib/modules/next.js

```diff
@@ -1,58 +1,54 @@
 export default (shower) => {
     const { stepSelector, activeSlideClass, visitedSlideClass } = shower.options;
+    const activeOrVisitedStepSelector = `${stepSelector}.${activeSlideClass}, ${stepSelector}.${visitedSlideClass}`;
```

Comment:
We can refactor this to `${stepSelector}:is(.${activeSlideClass}, .${visitedSlideClass})` as soon as `:is()` gains cross-browser support.